### ISSUE TITLE
"Could not parse JSON data"

### ISSUE THREAD

**Issue by HeSwartz** [code4recovery/pdf#83]
I'm trying to generate a PDF for a printed meeting book for just one town.  The PDF printer works fine for the entire list, but not for just one town.  When entering a URL for a region, for example, https://ct-aa.org/meetings/?tsml-day=any&tsml-region=guilford, I get the error message "Could not parse JSON data. Response was <html><title>You are being redirected...</title> <noscript>Javascript is required. Please enable jav…"  Java is enabled for all websites on Chrome.

**Comment by joshreisner** [comment]
looks like that's the URL of the web page - if you enter the URL of the JSON feed and then append `region=guilford` you should be good. do you have the JSON URL available?

**Comment by HeSwartz** [comment]
Thanks Josh. No, I don’t have the JSON URL available. Can you explain how I can get it? Thanks for your service. It’s admirable. Herb.

Sent from my iPhone

On Oct 22, 2024, at 2:18 PM, Josh Reisner ***@***.***> wrote:

﻿

looks like that's the URL of the web page - if you enter the URL of the JSON feed and then append region=guilford you should be good. do you have the JSON URL available?

**Comment by joshreisner** [comment]
looks like their feed is set to "open," so i'm sure they don't mind me sharing it publicly:

https://ct-aa.org/wp-admin/admin-ajax.php?action=meetings&region=guilford

**Comment by HeSwartz** [comment]
That worked.  Thanks.  You guys are the best!
________________________________

**Comment by HeSwartz** [comment]
Hi Josh. Is there anything we did the other day that could have caused this?

https://ct-aa.org/

Sent from my iPhone

**Comment by HeSwartz** [comment]
Not sure you got this.

[IMG_0040]

**Comment by anchovie91471** [comment]
Hey @HeSwartz! Yes, @joshreisner should see this, but I would recommend you start a discussion in the [TSML discussions](https://github.com/code4recovery/12-step-meeting-list/discussions) so it's tracked in the right place. I assume this latest question is in reference to your meetings page at https://ct-aa.org/meetings/?tsml-day=5&tsml-distance=10&tsml-mode=me?

**Comment by anchovie91471** [comment]
I do see one error on your meetings page that has to do with the WP Statistics plugin. You may want to try disabling that plugin temporarily to see if that fixes the problem.

**Comment by HeSwartz** [comment]
Thanks Anthony. Can you forward this to Josh?

Sent from my iPhone

On Oct 26, 2024, at 3:21 PM, Anthony Baggett ***@***.***> wrote:

﻿

I do see one error on your meetings page that has to do with the WP Statistics plugin. You may want to try disabling that plugin temporarily to see if that fixes the problem.

**Comment by HeSwartz** [comment]
Yes, That's the page.  We used the PDF generator for the first time and the site crashed few hours later.  Have there been any problems with the PDF generator?  My number is 203-654-6731.
________________________________
